include/seqan3/alignment/pairwise/execution/execution_handler_sequential.hpp

```diff
@@ -59,7 +60,36 @@ struct execution_handler_sequential
         static_assert(std::ranges::view<first_range_type>, "Expected a view!");
         static_assert(std::ranges::view<second_range_type>, "Expected a view!");
 
-        delegate(func(idx, std::move(first_range), std::move(second_range)));
+        delegate(algorithm(idx, std::move(first_range), std::move(second_range)));
+    }
+
+    /*!\brief Invokes the passed alignment instance in a blocking manner.
+     * \tparam fn_type           The callable that will be invoked.
+     * \tparam job_range_type    The type of underlying sequence pairs; must model std::ranges::view.
+     * \tparam delegate_type     The type of the callable invoked on the std::invoke_result of `fn_type.
+     *
+     * \param[in] algorithm    The callable which will invoked the alignment algorithm.
+     * \param[in] job_range    The range of underlying sequence pairs to be computed.
+     * \param[in] delegate     The callable which will be invoked with the result of the alignment.
```

Comment:
The function is invoked after the alignment algorithm and receives the alignment result as an argument, right? 
Please make more clear, what kind of function has to be passed here.